### PR TITLE
Ready crates publish

### PR DESCRIPTION
**Pull request by nicklan**
This gets repo ready to publish to crates:

names: delta_kernel, delta_kernel_derive, delta_kernel_ffi (we may want to rename `acceptance` at some point, but doesn't matter for now)
version: 0.0.1

### CODE REVIEW

File: ffi/Cargo.toml [hunk 1]
```diff
@@ -15,12 +15,13 @@ crate-type = ["cdylib", "staticlib"]
 [dependencies]
 tracing = "0.1"
 url = "2"
-deltakernel = { path = "../kernel", features = ["developer-visibility"] }
+delta_kernel = { path = "../kernel", features = ["developer-visibility"] }
 
 [build-dependencies]
 cbindgen = "0.26.0"
+delta_kernel = { path = "../kernel" }
```
Comment (zachschuermann) [inline ffi/Cargo.toml:22]: why build dep?

**Review by zachschuermann — APPROVED** [review]
one nit/idea: maybe workspace subdirectories should mirror crate names? e.g. rename `kernel` dir to `delta_kernel`?

File: ffi/Cargo.toml (hunk 1 above)
Comment (nicklan) [reply ffi/Cargo.toml]: oops, not sure how that snuck in. thanks.

**Comment by nicklan** [comment]
> one nit/idea: maybe workspace subdirectories should mirror crate names? e.g. rename `kernel` dir to `delta_kernel`?

We can do that yeah. it doesn't effect anything re publishing, so i'll say let's do that as a follow-up since it will change a ton of stuff

**Review by scarman-db — APPROVED** [review]

**Review by hntd187 — APPROVED** [review]

**Comment by zachschuermann** [comment]
sounds good just opened https://github.com/delta-incubator/delta-kernel-rs/issues/167 to remember :)

**Pull request merged**